### DISCUSSION TITLE
Classic Explorer Menu Bar (as in Win 7 and older)

### DISCUSSION THREAD

**Discussion by 59de44955ebd** [valinet/ExplorerPatcher#2633] [Ideas]
This screenshot shows how my Explorer in Win 11 (22H2, fully updated) currently looks like:
![win11_explorer](https://github.com/valinet/ExplorerPatcher/assets/12278038/957c73cd-a08c-4582-ba58-42e53456efc5)

About a year ago I had discovered that after using ExplorerPatcher to set the Control Interface to "Windows 10 Ribbon", I can use Winaero's [RibbonDisabler](https://winaero.com/ribbon-disabler-for-windows-10-and-windows-8/) (which officially is only meant for Win 8.x and 10) in a second step to replace the ribbon with the good old menu bar that I'm used to for about 25 years. This works just fine, I did not experience any unwanted side effects so far.

My question/suggestion: is there any chance to even integrate RibbonDisabler's functionality into ExplorerPatcher itself, so no second tool is needed? Since AFAIK RibbonDisabler is closed source, I'm not sure how it works, but I think it either patches or replaces explorerframe.dll in the system directory.

**Comment by tallman69** [comment]
In the Properties app for ExplorerPatcher go to the File Explorer tab and set the Control Interface to "Windows 7 Command Bar", and you will get your "good old menu bar"

**Comment by 59de44955ebd** [comment]
Damned, you are right! But I'm almost 100% certain that this option either didn't exist or didn't work as expected in my specific Win 11 system about a year ago, which is why I came up with this RibbonDisabler workaround. Anyway, thanks for the clarification, and sorry for the noise.
